### DISCUSSION TITLE
Aula (.NET/Estrutura do APP)

### DISCUSSION THREAD

**Discussion by cmsouzaac** [balta-io/2801#86] [Perguntas e Respostas] [answered]
Professor, 
Neste curso vamos aprender a trabalhar no ambiente de nível IDE (Visual Studio Code), entendo que o vscode é um ótimo editor chegando a quase ser uma IDE. Mais seria interessante trabalharmos em um ambiente focado realmente na tecnologia? essa é minha opinião, mais claro o senhor que possui a bagagem de sinalizar o que é melhor para nosso desenvolvimento profissional. Como ja deixei bem claro, sua didática é ótima e estou muito feliz em está no seu curso. Como muitos dizem o senhor é referencia no país na tecnologias MS.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia @cmsoouza, como vai?
Recomendo que comece pelo VS Code, ele te ajuda menos... nos próximos cursos vamos ver VS e Rider sim... mas no momento, foca no VS Code que vc vai aprender mais.

**Reply by cmsouzaac** [reply]
ok professor! Obrigado.
